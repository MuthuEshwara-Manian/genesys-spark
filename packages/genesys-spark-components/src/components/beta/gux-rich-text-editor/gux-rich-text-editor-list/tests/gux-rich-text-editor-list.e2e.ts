import { newSparkE2EPage, a11yCheck } from '../../../../../test/e2eTestUtils';

describe('gux-rich-text-editor-list', () => {
  [
    `<gux-rich-text-editor-list>
    <gux-rich-style-list-item text-style="heading-1" role="listitem" hydrated="">Heading 1</gux-rich-style-list-item>
    <gux-rich-style-list-item text-style="heading-2" role="listitem" hydrated="">Heading 1</gux-rich-style-list-item>
    <gux-rich-style-list-item text-style="heading-3" role="listitem" hydrated="">Heading 1</gux-rich-style-list-item>
    </gux-rich-text-editor-list>`
  ].forEach((html, index) => {
    it(`should display component as expected (${index + 1})`, async () => {
      const page = await newSparkE2EPage({ html });

      const element = await page.find('gux-rich-text-editor-list');

      await a11yCheck(page);

      expect(element.outerHTML).toMatchSnapshot();
    });
  });
});
