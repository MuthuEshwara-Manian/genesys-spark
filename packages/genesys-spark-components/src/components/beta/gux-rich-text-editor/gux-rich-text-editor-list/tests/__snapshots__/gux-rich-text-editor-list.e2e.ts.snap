// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`gux-rich-text-editor-list should display component as expected (1) 1`] = `"<gux-rich-text-editor-list role="list" hydrated=""><template shadowrootmode="open"><span tabindex="-1" aria-hidden="true"></span><slot></slot></template> <gux-rich-style-list-item text-style="heading-1" role="listitem" hydrated="">Heading 1</gux-rich-style-list-item> <gux-rich-style-list-item text-style="heading-2" role="listitem" hydrated="">Heading 1</gux-rich-style-list-item> <gux-rich-style-list-item text-style="heading-3" role="listitem" hydrated="">Heading 1</gux-rich-style-list-item> </gux-rich-text-editor-list>"`;
